### ISSUE TITLE
Add all Platforms available from IGDB

### ISSUE THREAD

**Issue by Iyagovos** [JosefNemec/PlayniteExtensions#190]
I'm using IGDB to catalogue every game I own, some of which are on multiple platforms. It'd be cool for the tool to be able to grab every platform that a game is available on from IGDB

**Comment by JosefNemec** [comment]
I don't think this makes sense since it would not reflect platforms you actually own. Plus some games have been released on ton of platforms with various ports so it would clutter platforms field a lot, in my opinion.
